### ISSUE TITLE
Login Problems when using Internet Explorer.

### ISSUE THREAD

**Issue by a1700** [owncloud/core#13436]
I'm %100 positive I'm using the correct password but soon as I click to login, the screen flashes at me and just presents the login screen again when using Internet Explorer. It doesn't even display the password reset link or anything. If I use Firefox everything works perfect.

**Comment by LukasReschke** [comment]
Thanks a lot for reporting issues back to us!

We need some more information to properly support you with your issue!

Can I ask you to follow our guidelines for submitting bugs as described here: https://github.com/owncloud/core/blob/master/CONTRIBUTING.md

Thanks a lot!

**Comment by MorrisJobke** [comment]
I just tested with IE8 and login works (tested with pressing enter and with clicking login). Also the reset password appears. I'm quite sure that this is caused by an app. Can you also specify the IE version you use ;)

**Comment by PVince81** [comment]
@a1700 does the login work if you enter your password a second time in IE ? (I remember seeing this before, usually after a redirect after a session timeout or something similar)

**Comment by PVince81** [comment]
If it doesn't work, try clearing the cookies and cache, reload the login page, then login again.

**Comment by a1700** [comment]
I was using OC 7.04 and Internet Explorer 11. I upgraded to OC 8 beta yesterday and everything is working OK now. I'll continue testing and let you know as soon as I have something new.

Thanks!!!

**Comment by myitbrain** [comment]
I've experienced the same before and I remember I cleaned cookies and cache and the same happened, but I think it was something wrong with my computer software, not with the ownCloud installation.

**Comment by PVince81** [comment]
Very strange, I just had it happening on 8.0.0beta1 just now with Chromium.
I had to clear cache + cookies and also clear the URL to avoid the redirect.

**Comment by jtremblay1977** [comment]
I have the same issue, but when i try with the ip, it's working.  I have that issue only with full name.

**Comment by TtuxX** [comment]
Hello @a1700 @PVince81 ! :)

I experimented this weird issue some time after upgrading from OC 7.0.4 to OC 8.0.0 

Symptoms were exactly the same as the first comment of both this issue thread and the first post of opentoe on the forum page (http://forum.owncloud.org/viewtopic.php?f=29&t=25769), except I had also no "remember" button on login page (I also could not log in and so on...).
Owncloud error messages were pretty much the same as opentoe reported on the forum: `"Unknown: POST Content-Length of 84 bytes exceeds the limit of 1 bytes at Unknown#0"`.
I did not experiment this problem on IE (I don't have it) and from what happened to me, it does not really seems to depend on the browser client at all (i've experimented the same behavior on Android, Chromium, Iceweasel...).

After searching a bit, I saw that the `N` number in the part `[...] exceeds the limit of N bytes at [...]` of the owncloud log error message was similar to the values I had in the owncloud `.htaccess` file, more precisely at both `upload_max_filesize` and `post_max_size` settings.

Mine was set on `2GB` for both, and the N in my error message was also `2`.
I realized that the max upload size you specify in OC administration page is now saved in owncloud's `.htaccess` file in bytes and that OC does not seems to see the `2GB` as gigabytes, but rather as `2` bytes.

If I am not mistaken, in version 7.0.*, upload sizes could be set under the layout of `2GB` instead of `2147483647` as it is the case for owncloud 8.0.0.
This looks like a retro-compatibility issue that has been introduced in ownCloud 8.0.0.

Is this a bug ? Can anyone confirm this behavior and see wether it is the true issue root or not ? If so, how do we handle this ?

Thanks ! :)
